src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,152 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private Node<K, V>[] nodesArray;
+    private int size = 0;
+    private int defaultCapacity = 16;
+    private double loadFactor = 0.75;
+
+    public MyHashMap() {
+        nodesArray = new Node[defaultCapacity];
+    }
+
+    public MyHashMap(int size) {
```

Comment:
what does this constructor do?

Reply:
It creates a new empty HashMap with specified capacity.